### PR TITLE
infra(arch): Add initial architecture documentation template

### PR DESCRIPTION
**Pull request by blaumeiser-at-bosch**
Signed-off-by: Lars Geyer-Blaumeiser <lars.geyer-blaumeiser@bosch-si.com>

### CODE REVIEW

**Comment by blaumeiser-at-bosch** [comment]
Starting point for #163

**Comment by maxhbr** [comment]
maybe merge this into `/antenna-documentation/`, e.g. into `/antenna-documentation/src/arc42/`

**Comment by blaumeiser-at-bosch** [comment]
> maybe merge this into `/antenna-documentation/`, e.g. into `/antenna-documentation/src/arc42/`

@maxhbr: I though antenna-documentation is generating a webpage as user documentation. I am not sure whether the architecture documentation should be part of a user docu.

**Comment by maxhbr** [comment]
missing license header in `architecture-docu/antenna.plantuml` (or should we blacklist `*.platuml` for the license header check?)

**Comment by blaumeiser-at-bosch** [comment]
Solves https://github.com/bsinno/osm-antenna/issues/488

File: architecture-docu/2019_06_25_DepTree.txt [hunk 1]
```diff
@@ -0,0 +1,219 @@
+[INFO] ------------------< org.eclipse.sw360.antenna:model >-------------------
```
Comment (sschuberth) [inline architecture-docu/2019_06_25_DepTree.txt:1]: Should this file really be part of the PR?

**Comment by sschuberth** [comment]
@blaumeiser-at-bosch is this PR still active / relevant?

**Pull request closed without merging**